apps/examples/testcase/le_tc/network/tc_net_ioctl.c

```diff
@@ -80,8 +80,7 @@ static void tc_net_ioctl_fionread_n(void)
 		return;
 	}
 	long a = 10;
-
-	int ret = ioctlsocket(fd, FIONREAD, &a);
+	int ret = ioctl(fd, FIONREAD, (unsigned long)&a);
```

Comment:
How about declaring the type of the variable 'a' as unsigned long instead of just long in line 82?

Reply:
ok